tests/e2e/gc_block_volume_expansion.go

```diff
@@ -1828,6 +2127,47 @@ func onlineVolumeResizeCheck(f *framework.Framework, client clientset.Interface,
 	// first time
 	gomega.Expect(fsSize).Should(gomega.BeNumerically(">", originalSizeInMb), fmt.Sprintf("error updating filesystem size for %q. Resulting filesystem size is %d", pvclaim.Name, fsSize))
 	ginkgo.By("File system resize finished successfully")
+
 	framework.Logf("Online volume expansion in GC PVC is successful")
 
 }
+
+func createStaticPVandPVCinGuestCluster(client clientset.Interface, ctx context.Context, namespace string, svpvcName string, size string, storageclass *storagev1.StorageClass) (*v1.PersistentVolumeClaim, *v1.PersistentVolume, *v1.Pod, string) {
+	ginkgo.By("Creating PV in guest cluster")
+	gcPV := getPersistentVolumeSpecWithStorageclass(svpvcName, v1.PersistentVolumeReclaimDelete, storageclass.Name, nil, size)
```

Comment:
better to match with reclaim policy of SVC PVC IMO

Reply:
statically created PVC will default get reclaim policy as delete in SVC namespace. Hence I used delete here also

Reply:
Please add a comment here which mentions the same.